code/workspaces/web-app/src/components/welcome/VerifyEmail.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import datalabsLogo from '../../assets/images/datalabs-vert.png';
+import PagePrimaryActionButton from '../common/buttons/PagePrimaryActionButton';
+
+const styles = theme => ({
+  bar: {
+    backgroundColor: theme.palette.backgroundDark,
+    textAlign: 'center',
+    zIndex: 2,
+    padding: `${theme.spacing(14)}px ${theme.spacing(2)}px`,
+  },
+  logo: {
+    height: 300,
+  },
+  tagLine: {
+    color: theme.palette.secondary[50],
+    padding: 20,
+  },
+  buttons: {
+    display: 'flex',
+    justifyContent: 'center',
+    margin: `${theme.spacing(4)}px ${theme.spacing(1)}px`,
+    marginBottom: 0,
+  },
+  button: {
+    '& + &': {
+      marginLeft: theme.spacing(2),
+    },
+  },
+});
+
+const tagLine = `Before logging in you need to verify your e-mail address. Please use the
+  link in the e-mail which will be sent to you shortly. Note that you may have to check 
+  your spam folder.`;
+
+const VerifyEmail = ({ classes }) => (
+  <div className={classes.bar}>
+    <img className={classes.logo} src={datalabsLogo} alt="DataLabs-Logo" />
+    <Typography className={classes.tagLine} variant="h6">{tagLine}</Typography>
```

Comment:
This renders as all capitals, which I found a bit 'shouty'?

Reply:
I pointed Iain to the cover from the public app to base it on which is where the text style comes from. I agree that with the extra text it becomes a bit "shouty". I had a little play with the styling in some spare time I had the other day and altered it to the attached image. I have it as code changes so if people like it or want to take it forward as a base then I can merge it into this branch.

![image](https://user-images.githubusercontent.com/52826110/95864548-6dc9e680-0d5d-11eb-920b-b1732ad3a493.png)

Reply:
Yes that would be great if you don't mind